gulpfile.js

```diff
@@ -108,37 +108,12 @@ gulp.task('angular-chosen-img', function () {
         .pipe(gulp.dest(path.join(output_directory, 'css')));
 });
 
-// Catches all skins files for TinyMCE editor.
-gulp.task('tinymce-skins', function () {
-    return gulp.src(path.join('bower_components', 'tinymce', 'skins', '**'))
-        .pipe(gulp.dest(path.join(output_directory, 'tinymce', 'skins')));
+// Extra task only for CKEditor
```

Comment:
But what does this task do? Why do we have to copy these files to the output directory *and* catch all these files in the bower.json main section?